### PR TITLE
TC_14.002.15_A| Header > Search Box | Verify a User can select a suggestion to auto-fill the search box and complete the search

### PR DESCRIPTION
**Pull request by LanaUno**
[TC_14.002.15](https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/293)

### CODE REVIEW

File: cypress/e2e/searchBoxAutoFill.cy.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    cy.get('.org_jenkinsci_plugins_workflow_job_WorkflowJob > :nth-child(2) > .desc').click()
+    cy.get('#ok-button').contains("OK").click();
+    cy.get(".jenkins-submit-button").click({force:true});
+    cy.get("#jenkins-name-icon").click();
+    cy.get('#search-box').type('Pro')
+    cy.get('#search-box-completion').contains("Project TC_14.002.15_A").click()
+    cy.get('#search-box').type('{enter}')
+    cy.get('.job-index-headline').should('have.text', 'Project TC_14.002.15_A')
+    })
+})
```
Comment (adiuzhikov) [inline cypress/e2e/searchBoxAutoFill.cy.js:21]: We already possess the specification file for this US - headerSearchBox.cy.js. Please compose your test within that file. Additionally, ensure that your test is properly aligned.

**Review by adiuzhikov — CHANGES_REQUESTED** [review]

File: cypress/e2e/searchBoxAutoFill.cy.js (hunk 1 above)
Comment (LanaUno) [reply cypress/e2e/searchBoxAutoFill.cy.js:21]: > We already possess the specification file for this US - headerSearchBox.cy.js. Please compose your test within that file. Additionally, ensure that your test is properly aligned

Ok, but how I can do this?

Comment (LanaUno) [reply cypress/e2e/searchBoxAutoFill.cy.js:21]: I deleted previous file [searchBoxAutoFill.cy.js], added my test to  headerSearchBox.cy.js. Tests passed locally.

**Review by Olena-Novikova — COMMENTED** [review]
Super!

File: cypress/e2e/searchBoxAutoFill.cy.js (hunk 1 above)
Comment (LanaUno) [reply cypress/e2e/searchBoxAutoFill.cy.js:21]: After such a long time waiting for review this branch has conflicts which I can't resolve by myself, I heed help

Comment (LanaUno) [reply cypress/e2e/searchBoxAutoFill.cy.js:21]: > > After such a long time waiting for review this branch has conflicts which I can't resolve by myself, I heed help
> 
> Consider utilizing Nalya’s review, which demonstrates a step-by-step resolution of conflicts. Additionally, I noticed that you have not deleted the newly created file. Kindly delete the file and align your test accordingly.

Conflict is resolved, could you please approve PR?

Comment (adiuzhikov) [reply cypress/e2e/searchBoxAutoFill.cy.js:21]: Once more, you are editing the incorrect file (a newly created one). Please delete that file and make all changes with your test in the existing one. Kindly adhere to the instructions provided.

**Review by adiuzhikov — APPROVED** [review]

**Review by Olena-Novikova — COMMENTED** [review]
Good job.

**Review by Olena-Novikova — APPROVED** [review]

**Pull request merged**